go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/kubernetes-csi/csi-lib-utils
+
+go 1.12
+
+require (
```

Comment:
In contrast to the manually written `Gopk.toml`, `go.mod` is complete and lists all dependencies with revisions chosen at the time of running `go mod init`. We have less control over dependency resolution (can't specify the exact minimum version that the code is compatible with, can't specify ranges), but that's simply how "go mod" works. :man_shrugging:

Reply:
A diff of `vendor` shows that we still use the same source code. Just pruning is handled differently, so some files are gone and others are new.

Compilation is done without Go modules, so nothing changes when building.